open-company/open-company-web
# Dynamic

Card: https://trello.com/c/DjBjbw17

To test:
- connect with 2 different accounts to the same board (with author permissions at least)
- add a topic from one
- open the topic view on both accounts
- comment on the entry from the first account
- the other user sees the new comment? good
- add a topic and comment on that from the first account
- the other user sees the new entry with the right count of comments? good
- add a reaction for one of the entry from the first account
- the other user sees the reaction? good
- open another tab with the same session of the first account and navigate to the same topic view
- add another reaction from the first tab
- check that the second tab sees the reaction and it's coming from the self user (the reaction is blue)
